### PR TITLE
SBT-web back ref explaining usage

### PR DESCRIPTION
**Pull request by huntc**

### CODE REVIEW

**Comment by joescii** [comment]
Good call.  I actually was referring to the headline which is super easy to forget even exists once you're used to seeing a project. :)

_SBT plugin for running mocha JavaScript unit tests on node_

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ sbt-mocha
 
 [![Build Status](https://api.travis-ci.org/sbt/sbt-mocha.png?branch=master)](https://travis-ci.org/sbt/sbt-mocha)
 
-Allows mocha to be used from within sbt.
+Allows mocha to be used from within sbt via the use of [sbt-web](https://github.com/sbt/sbt-web). You can therefore run your JS tests using SBT conventions either in-JVM (eg Tririme), or out (Node).
```
Comment (xuwei-k) [inline README.md]: s/Tririme/Trireme ?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ sbt-mocha
 
 [![Build Status](https://api.travis-ci.org/sbt/sbt-mocha.png?branch=master)](https://travis-ci.org/sbt/sbt-mocha)
 
-Allows mocha to be used from within sbt.
+Allows mocha to be used from within sbt via the use of [sbt-web](https://github.com/sbt/sbt-web). You can therefore run your JS tests using SBT conventions either in-JVM (eg Trireme), or on (Node).
```
Comment (benmccann) [inline README.md]: I don't think you can run on Trireme anymore

Comment (jroper) [reply README.md]: npm won't run on Trireme, but mocha I think should.

Comment (benmccann) [reply README.md]: We're getting lots of bug reports right now of sbt-less being broken if you don't have node installed. I don't have time to investigate, but it kind of looked like sbt-web in general was broken with Trireme

**Pull request merged**